src/content/docs/accounts/accounts-billing/new-relic-one-user-management/authentication-domains-saml-sso-scim-more.mdx

```diff
@@ -77,7 +77,7 @@ From the [authentication domain UI](#ui), you can set one of two options for how
 * **Manual:** This means that your users are added manually to New Relic from the [**User management** UI](/docs/accounts/accounts-billing/new-relic-one-user-management/add-manage-users-groups-roles/#where). 
 * **SCIM:** Our automated user management (AUM) feature allows you to use SCIM provisioning to import and manage users from your identity provider. For instructions, see [Automated user management](/docs/accounts/accounts/automated-user-management/automated-user-provisioning-single-sign). 
 
-Note that once you enable SCIM for an authentication domain, you can't later turn it off. Instead, you should create a new one. 
+Note that once you enable SCIM for an authentication domain, you can't later turn it off. If you need an authentication domain without SCIM, you must create a new one. 
```

Comment:
Comment from the sidelines: Is it clear to the target audience what challenges this could cause? (Since I know very little about SCIM, I'm clearly not the target audience)